neptune/new/handler.py

```diff
@@ -447,14 +447,14 @@ def track_files(self, path: str, destination: str = None, wait: bool = True) ->
 
             if not attr:
                 attr = Artifact(self._run, parse_path(self._path))
-                attr.track_files_to_new(
+                attr.track_files(
                     project_uuid=self._run._project_uuid,  # pylint: disable=protected-access
                     source_location=path,
                     destination=destination,
                     wait=wait
                 )
             else:
-                attr.track_files_to_existing(
+                attr.track_files(
```

Comment:
This is exactly same call as above.

Reply:
And it should be called after `self._run.set_attribute`. Please merge current master to this branch and rebase this PR.